pkg/prometheus/testdata/ServiceMonitorObjectWithMatchExpressionSelector.golden

```diff
@@ -0,0 +1,78 @@
+global:
+  scrape_interval: 30s
+  external_labels:
+    prometheus: default/test
+    prometheus_replica: $(POD_NAME)
+  evaluation_interval: 30s
+scrape_configs:
+- job_name: serviceMonitor/default/defaultServiceMonitor/0
+  honor_labels: false
+  kubernetes_sd_configs:
+  - role: endpoints
+    namespaces:
+      names:
+      - default
+    selectors:
+    - role: endpoints
```

Comment:
should we also add a service role selector?
```suggestion
    - role: service
      label: group in (group1)
    - role: endpoints
```

Reply:
Actually I'm not sure, I'd say no.
But wdyt? @simonpasquier Do you have any nice use case?